scripts/build/download-tool.sh

```diff
@@ -78,7 +78,7 @@ if [ -f "$ARGV_OUTPUT" ]; then
   fi
 fi
 
-wget --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT"
+wget --no-verbose --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT" 2>&1
```

Comment:
When we're doing local builds, seeing the verbose output is actually useful. Is there some envvar we can check for to see if we're running on a CI server, and use that to change the command-args as appropriate?
(and why is the `2>&1` necessary?)